dev/cloud9-generator/Makefile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+pretty:
+	prettier --no-config --write lib/**/*.ts
+	prettier --no-config --write bin/*.ts
+	prettier --no-config --write tsconfig.json
+
+clean:
+	find bin/ lib/ -name '*.js' -exec rm -v {} \;
+	find bin/ lib/ -name '*.d.ts' -exec rm -v {} \;
+
+cloud9:
+	@echo "Development tool only! Workshop students do not need to run this target."
+	@# synth does not catch if build errors have been introduced
+	@npm run build
+	cdk synth BusyEngineersCloud9Stack > ./document-bucket-cloud9-bootstrap.yaml
+	aws s3 cp ./document-bucket-cloud9-bootstrap.yaml s3://busy-engineers-cfn --acl public-read
```

Comment:
More importantly, it looks like workshop students *cannot* run this target since they won't have write permissions to this bucket.

Reply:
Well, they can run the target, but it will fail.

It's not a problem if they do, it just won't do anything.

Hence the language of "not need" -- a student does not need to worry about the error message, and nothing was harmed if they tried.